clipboard-extension.md

```diff
@@ -0,0 +1,44 @@
+# Clipboard Extension
+
+The Clipboard Extension supports copying to the clipboard using a subset of the XTerm OSC 52 escape sequence. 
+
+## Feature detection
+
+Support for this functionality in a terminal is indicated by the extension parameter 52 in the Device Attributes (DA) report.
+
+## Syntax
+The escape sequence has the following syntax:
+
+```
+OSC 52 ; Pc ; Pd ST
+```
+
+Conforming terminals MUST at least support an empty Pc value, and a Pc value of c. Other values are optional.
+
+Conforming terminals MUST support a Pd value that is a base64 encoded representation of the content that will be copied to the clipboard. They SHOULD also support an empty Pd value to clear the clipboard. They MAY support a Pd value of ? to query the clipboard content (as specified in the XTerm documentation).
+
+If a terminal has an option to disable clipboard access, it MUST only indicate support for this extension when writing to the clipboard is actually allowed.
+
+### Adoption State
+
+| Support | Terminal/Tookit/App                                        | Notes |
+|---------|------------------------------------------------------------|-------|
+| not yet | [Alacritty](https://github.com/alacritty/alacritty)        | |
+| not yet | [Contour](https://github.com/contour-terminal/contour/)    | [contour#1765](https://github.com/contour-terminal/contour/issues/1765) |
+| not yet | [DomTerm](https://github.com/PerBothner/DomTerm)           | [doomterm#124](https://github.com/PerBothner/DomTerm/issues/124) |
```

Comment:
The issue link in the notes should assumedly be `[domterm#124]` (not **doom**term).